assets/source/setup-guide/app/steps/SetupTracking.js

```diff
@@ -218,9 +225,66 @@ const SetupTracking = ( { view } ) => {
 
 			setIsSaving( false );
 		},
-		[ appSettings, setIsSaving, setAppSettings, createNotice ]
+		[
+			appSettings,
+			setIsSaving,
+			setAppSettings,
+			connectAdvertiser,
+			createNotice,
+		]
 	);
 
+	const connectAdvertiser = useCallback( async () => {
+		setIsFetching( true );
+
+		try {
+			const results = await apiFetch( {
+				path:
+					wcSettings.pinterest_for_woocommerce.apiRoute +
+					'/tagowner/?advrtsr_id=' +
+					appSettings.tracking_advertiser +
+					'&tag_id=' +
+					appSettings.tracking_tag,
+				method: 'GET',
```

Comment:
To "connect advertiser" is to _create a connection_ to the advertiser, so it should be a `POST` method.